rocAL/source/augmentations/node_ssd_random_crop.cpp

```diff
@@ -129,7 +129,7 @@ void SSDRandomCropNode::update_node() {
                 crop_box.b = 1;
                 break;
             }
-            float aspect_ratio;
+            float aspect_ratio = 0.0f;
```

Comment:
aspect_ratio should be 1.0 by default

Reply:
The aspect_ratio is initialized to 0.0 since it has to be computed in the code below.
This particular algorithm has a [condition ](https://github.com/ROCm/rocAL/blob/235e493fc258ee02c55400421298e910134a0631/rocAL/source/augmentations/node_ssd_random_crop.cpp#L142)to check if the generated aspect ratio falls between 0.5 and 2 and it proceeds with the remaining code, if it does not satisfy the condition it would regenerate the aspect ratio value, Hence we have used 0.0 over here.